.github/workflows/build_and_test.yml

```diff
@@ -38,23 +39,29 @@ jobs:
       if: github.event_name == 'push'
       run: |
         ./.github/workflows/build-test nomypy
-      env:
-        MODULE: ${{ matrix.module }}
     - name: Set up Python 3.9
       if: github.event_name == 'pull_request' || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel')
       uses: actions/setup-python@v3
       with:
         python-version: '3.9'
-    - name: Build and test including remote checks (3.9)
+    - name: Build and test including remote checks (3.9) mypy
```

Comment:
Looks like both the mypy and nomypy versions have been run on all platforms?

Reply:
Thank you, should be fixed now. @cqc-melf
[fix workflow and version in tests](https://github.com/CQCL/pytket-quantinuum/pull/3/commits/c3b9fa3e6f4c30aae59efb1492b6e80d9c5f101a)